src/OSPSuite.Core/Domain/Builder/ModelConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace OSPSuite.Core.Domain.Builder
+{
+   public class ModelConfiguration
```

Comment:
I realized we were passing the two parameters ALL The time together. I created a small helper object to make all of this more ergonomic.